### ISSUE TITLE
Nopol fails to compile due to the inaccessibility of the "Maven Repository for Spoon Release"

### ISSUE THREAD

**Issue by DehengYang** [SpoonLabs/nopol#225]
Sorry to interrupt here. I am writing to ask for your help in compiling Nopol. 

I had a `(connection time out)` error when compiling Nopol. After taking a closer look at its pom.xml (see https://github.com/SpoonLabs/nopol/blob/master/nopol/pom.xml), I observed that the repository (see https://github.com/SpoonLabs/nopol/blob/8f587440aee4b5ecddfbfd3198f9c784bef19286/nopol/pom.xml#L307) could not be visited... Its URL is: http://sachaproject.gforge.inria.fr/repositories/releases/

It would be much appreciated if any help could be provided. Thank you!

**Comment by monperrus** [comment]
Thanks for the bug report, looking at it.

**Comment by monperrus** [comment]
might be fixed by https://github.com/SpoonLabs/nopol/pull/226

**Comment by DehengYang** [comment]
Thank you so much for the timely guidance and help. I can package it successfully now. 

I found that the reason for the connection time out error when running `mvn package` may be caused by the network issue rather than the nopol issue. In my region, some remote repositories cannot be reached sometimes. I am sorry for misreporting the issue.

Seemingly there is no need to change the `http` into `https`, as I can now download the jar from both sites.

+ With HTTP:
> Downloading: http://sachaproject.gforge.inria.fr/repositories/releases/gov/nasa/jpf/jpf-symbc/576/jpf-symbc-576.pom
> [WARNING] Could not validate integrity of download from http://sachaproject.gforge.inria.fr/repositories/releases/gov/nasa/jpf/jpf-symbc/576/jpf-symbc-576.pom: Checksum validation failed, no checksums available
> [WARNING] Checksum validation failed, no checksums available for http://sachaproject.gforge.inria.fr/repositories/releases/gov/nasa/jpf/jpf-symbc/576/jpf-symbc-576.pom
> Downloaded: http://sachaproject.gforge.inria.fr/repositories/releases/gov/nasa/jpf/jpf-symbc/576/jpf-symbc-576.pom (463 B at 0.0 KB/sec)
> Downloading: http://sachaproject.gforge.inria.fr/repositories/releases/gov/nasa/jpf/jpf-symbc/576/jpf-symbc-576.jar

+ With HTTPS:
> Downloading: https://sachaproject.gforge.inria.fr/repositories/releases/gov/nasa/jpf/jpf-symbc/576/jpf-symbc-576.pom
> [WARNING] Could not validate integrity of download from https://sachaproject.gforge.inria.fr/repositories/releases/gov/nasa/jpf/jpf-symbc/576/jpf-symbc-576.pom: Checksum validation failed, no checksums available
> [WARNING] Checksum validation failed, no checksums available for https://sachaproject.gforge.inria.fr/repositories/releases/gov/nasa/jpf/jpf-symbc/576/jpf-symbc-576.pom
> Downloaded: https://sachaproject.gforge.inria.fr/repositories/releases/gov/nasa/jpf/jpf-symbc/576/jpf-symbc-576.pom (463 B at 0.1 KB/sec)
> Downloading: https://sachaproject.gforge.inria.fr/repositories/releases/gov/nasa/jpf/jpf-symbc/576/jpf-symbc-576.jar

Although the website https://sachaproject.gforge.inria.fr/repositories/releases/ is still not accessible, interestingly, the http://sachaproject.gforge.inria.fr/repositories/releases/gov/nasa/jpf/jpf-symbc/576/jpf-symbc-576.pom is accessible and can be downloaded by maven.

Thank you again for the great patience and help.
